test/torchaudio_unittest/README.md

```diff
@@ -9,6 +9,7 @@ For testing, the following packages are needed:
 
 ```bash
 pip install typing pytest scipy numpy parameterized
+pip install -r requirements.txt
```

Comment:
Instead adding duplicated information, can you replace the whole section (from `For testing, the following ...`) with a redirection to `CONTRIBUTING.md`? In all the `help-wanted` issues, we tell contributors to follow the instruction in `CONTRIBUTING.md`.

Reply:
I've addressed this here https://github.com/pytorch/audio/pull/1584/files#diff-937f67c5380ec39c13ab0d069d3dff30ec219a3fe7c8725605caa95120b65e0cR8